content/efcore/getting-started/samples/loading-related-data-tutorial/multiple-include-methods/Program.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var book = context.Books
+                .Include(b => b.Editions)
+                .Include(b => b.Author)
+                .Single(b => b.BookId == 1);
+
+            foreach (Edition edition in book.Editions)
+            {
+                Console.WriteLine("{0} - {1} {2}", edition.Book.Title, book.Author.FirstName, book.Author.LastName);
```

Comment:
I guess we do access the information through a different route which shows the editions are loaded. Maybe we could add a comment to say like - hey, it's printing the same info, but we can see that the editions are also loaded because we're using them to show the same stuff?